tests/neo.UnitTests/IO/Caching/UT_CloneCache.cs

```diff
@@ -35,6 +35,7 @@ public void TestAddInternal()
 
             cloneCache.Commit();
             myDataCache[new MyKey("key1")].Should().Be(new MyValue("value1"));
+            cloneCache.Commit();
```

Comment:
It's required? maybe you should test the `GetChanges` after this

Reply:
It's not required. This line should be removed.

Reply:
Have removed these changes